databases/week1/Homework-week-1.sql

```diff
@@ -0,0 +1,42 @@
+-- 1. Find out how many tasks are in the task table
+SELECT COUNT(*) FROM task;
+
+-- 2. Find out how many tasks in the task table do not have a valid due date
```

Comment:
Just a tiny comment, the more standard way to write queries is to have the `SELECT`, `FROM`, and `WHERE` in different lines, for instance:

```
SELECT COUNT(*) 
FROM task
WHERE due_date IS null;
```